connectors/ftp-connector/src/test/scala/zio/connect/ftp/FtpConnectorSpec.scala

```diff
@@ -0,0 +1,210 @@
+package zio.connect.ftp
+
+import zio._
+import zio.connect.ftp.FtpConnector.PathName
+import zio.stream.ZPipeline.utf8Decode
+
+import zio.stream.ZStream
+import zio.test.Assertion._
+import zio.test._
+
+import java.io.IOException
+
+trait FtpConnectorSpec extends ZIOSpecDefault {
```

Comment:
Suites should be ordered alphabetically in implementation and in the composed result